pages/common/gh-pr-merge.md

```diff
@@ -1,32 +1,28 @@
 # gh pr merge
 
-> Merge GitHub pull requests.
+> Merge a GitHub pull request.
 > More information: <https://cli.github.com/manual/gh_pr_merge>.
 
 - Merge the pull request associated with the current branch interactively:
 
 `gh pr merge`
 
-- Merge the specified pull request, interactively:
+- Merge with a merge commit:
```

Comment:
This could use some more explanation. I don't know but I'm assuming this is correct
```suggestion
- Merge the current branch into the specified pull request:
```